### ISSUE TITLE
Cant Install or Download CIA

### ISSUE THREAD

**Issue by Arktoja** [llakssz/CIAngel#134]
I'm constantly recieving an error:

-One or both of the signatures in  the CETK 'Cert Chain' are unrecognized
-The TMD signature could not be recognized
-Input files could no be processed successfully

Could not install the CIA

**Comment by DrakiaXYZ** [comment]
Sounds like you have Nintendo's servers blocked

**Comment by rhynoceros** [comment]
Hello I'm getting the exact same error. Today I've installed Luma 8.1 on two different O3NDS. Both same firmware pre and post CFW. For some reason on one 3DS I can install CIA's without any problems. On the other one I'm getting this error. Hence why I assume it's not Nintendo servers blocked for me?

Hope u can find time to help me out. Very thankful for your work.

**Comment by DrakiaXYZ** [comment]
Verify you aren't running custom DNS on one of the 3DS'. The only cause of this error that I know of is Nintendo's servers being unreachable.

**Comment by rhynoceros** [comment]
Thanks for your reply! I checked and on both 3DS' the DNS setting is on auto-obtain. I tried using Google DNS but for some reason I can't input custom dns.

**Comment by DrakiaXYZ** [comment]
Outside of internet issues, or perhaps some weird Luma setting (Maybe the devunit setting?), I'm unsure why you would be getting these errors.

**Comment by rhynoceros** [comment]
Could you maybe elaborate? I don't know much about the hacking itself. I run the same settings in luma config and same .cia's installed.

**Comment by rhynoceros** [comment]
I was doing something wrong with the DNS. It's on Google DNS now, I've used a small size game to test and it installed correctly. Now trying with a triple AAA game. Will keep updated.

**Comment by DrakiaXYZ** [comment]
> Could you maybe elaborate?

Essentially, there's no reason within the app for this, it has to be something on the 3DS itself, or internet related (DNS, Luma settings, etc), but that's outside my ability to debug.

**Comment by rhynoceros** [comment]
I see what you're aaying, thank you. Also I just tried Majora's Mask, which installed effortlessly on the other 3DS, but again gave an error on this one. No clue what could be causing this.

**Comment by urherenow** [comment]
just curious, do both systems have the SAME version of b9s installed and SAME commit of Luma? Just wondering if the custom sighax stuff could possibly mess with verifying legit certs or not. I'm not too knowledgeable on the subject...

**Comment by rhynoceros** [comment]
Yeah I think so. I hacked both DS' yesterday afternoon with the same files.

**Comment by bigretromike** [comment]
`Sounds like you have Nintendo's servers blocked` @TheDgtl which servers that would be? which one we need for this to work ?

**Comment by DrakiaXYZ** [comment]
@bigretromike No idea off the top of my head. I recommend not blocking anything if you want to download games from their servers. There's a good chance it's just these two, unless they do any redirects:
ccs.cdn.c.shop.nintendowifi.net
kagiya-ctr.cdn.nintendo.net
